clusterous/cluster.py

```diff
@@ -784,6 +785,20 @@ def _create_route_table(self, vpc_conn, vpc, route_table_name):
     
         return route_table
 
+    def _get_user_data(self, playbook, vars):
+        with io.BytesIO() as fw:
+            fw.write("#!/bin/bash\ncat > /tmp/{0} <<- EOM\n".format(playbook))
+            playbook_content = open(defaults.get_script('ansible/remote/{0}'.format(playbook)),'r').read()
+            for k,v in vars.iteritems():
+                playbook_content = playbook_content.replace(k,v)
```

Comment:
Substituting values in the Ansible script using string.replace() could be a bit unreliable. Ansible uses jinja2 for templating, so if anything that would be better.

However, I don't think it's necessary to substitute the values here. The variables could just go into a separate yaml file, and then pass that file in to Ansible when running the playbook.